netlify.toml

```diff
@@ -3,4 +3,4 @@
 [build]
   base = "code/"
   publish = "build/"
-  command = "npm run build"
+  command = "CI=false && npm run build"
```

Comment:
⭐
What errors do you get though?

Reply:
Hade to put that command there to be able to deploy at all, probably due to previous vulnerabilities that prevented us from deploying, since then changes have been made so the issues are resolved now.